lib/rack/constants.rb

```diff
@@ -16,13 +16,22 @@ module Rack
   HTTP_COOKIE       = 'HTTP_COOKIE'
 
   # Response Header Keys
-  CACHE_CONTROL     = 'cache-control'
-  CONTENT_LENGTH    = 'content-length'
-  CONTENT_TYPE      = 'content-type'
-  ETAG              = 'etag'
-  EXPIRES           = 'expires'
-  SET_COOKIE        = 'set-cookie'
-  TRANSFER_ENCODING = 'transfer-encoding'
+  ACCEPT_ENCODING     = 'accept-encoding'
+  ACCEPT_RANGES       = 'accept-ranges'
+  ALLOW               = 'allow'
+  CACHE_CONTROL       = 'cache-control'
+  CONTENT_ENCODING    = 'content-encoding'
+  CONTENT_DISPOSITION = 'content-disposition'
+  CONTENT_LENGTH      = 'content-length'
+  CONTENT_RANGE       = 'content-range'
+  CONTENT_TYPE        = 'content-type'
+  ETAG                = 'etag'
+  EXPIRES             = 'expires'
+  LAST_MODIFIED       = 'last-modified'
+  SET_COOKIE          = 'set-cookie'
+  TRANSFER_ENCODING   = 'transfer-encoding'
+  VARY                = 'vary'
+  X_CASCADE           = 'x-cascade'
```

Comment:
Adding new ones in alphabetical order, but not reordering what was already out of order.